docs/wiki/Home.md

```diff
@@ -47,20 +50,23 @@ If you're unfamiliar with Infrastructure as Code or wonder how you can use the c
 
 # Scope
 
-Following, you can find an abstract overview of everything in- and out-of-scope of this repository.
+In next section, you can find an overview of what is in scope and what is out of scope for CARML.
 
 ## In Scope
 
-- **Module library:** A collection of comprehensive, reusable, Bicep-based building blocks to deploy Azure resources, that can be combined to create & orchestrate more complex, multi-module Azure solutions.
-- **CI environment:** Pipelines to validate modules & publish the successfully validated ones to a target location, using semantic versioning. Available with both GitHub Workflows and Azure DevOps pipelines.
-- **Documentation:** Including design principles and usage of [The library](./The%20library) and [The CI environment](./The%20CI%20environment) as well as step-by-step guidelines on how to start leveraging them - see in the [Getting started](./Getting%20started) section.
+- **Module library** is a collection of comprehensive, reusable, Bicep-based building blocks to deploy Azure resources. It can also be combined to create & orchestrate more complex, multi-module Azure solutions.
+- **CI environment** is DevOps pipelines. It can be used to validate modules and to publish modules which are successfully validated to a target location. It uses semantic versioning and it can be used with both GitHub workflows and Azure DevOps pipelines.
```

Comment:
```suggestion
- **CI environment:** it provides DevOps pipelines. It can be used to validate modules and to publish modules which are successfully validated to a target location. It uses semantic versioning, and it can be used with both GitHub workflows and Azure DevOps pipelines.
```